### PR TITLE
Update discordBot.md to disable public bot.

### PR DESCRIPTION
**Pull request by fewtime**
There's an error(as follows) while I'm trying to create a private bot.  After adding the step 3 can solve the problem.

Error: Private application cannot have a default authorization link.

### CODE REVIEW

**Comment by fewtime** [comment]
I accidentally deleted the original step 3, and then resubmitted it after adding it.

File: guides/discordBot.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
-2. Enter a name for your bot, like `AutoRSA` and click `Create`.
-3. Click on `Bot` in the left sidebar, then give it a name and profile picture if you want.
-4. Disable `Public Bot`.
-5. Under `Privileged Gateway Intents`, enable `Message Content Intent`.
-6. Click on `OAuth2` in the left sidebar, then `URL Generator`. Then scroll down to `OAuth2 URL Generator`. 
-7. Under `Scopes` select `bot`. Then underneath that in `Bot Permissions` select `Send Messages` and `Read Message History`.
-8. Copy the link in the `Scopes` section and paste it into your browser. Select the server you want to add the bot to and click `Authorize`.
-9. Click on `Bot` in the left sidebar. Under `Token`, click `Reset Token`. Copy the new token and paste it into your `.env` file as `DISCORD_TOKEN`.
-10. To get the Channel ID, go to `Advanced` in Discord settings, then turn on `Developer Mode`. Then right click on the channel for the bot and click `Copy ID`. Paste the ID into your `.env` file as `DISCORD_CHANNEL`. If you want to turn off `Developer Mode`, you can do so, but it isn't necessary.
+2. Enter a name for your bot, like `AutoRSA` and click `Create.
```
Comment (NelsonDane) [inline guides/discordBot.md:5]: This line appears unchanged (except for the missing tick on create). Can you put that back?

**Comment by NelsonDane** [comment]
Thanks for the PR, give me a little bit of time to test this :)

**Review by NelsonDane — CHANGES_REQUESTED** [review]

File: guides/discordBot.md (hunk 1 above)
Comment (NelsonDane) [reply guides/discordBot.md:5]: @fewtime Do you still want me to review this?

Comment (fewtime) [reply guides/discordBot.md]: Sorry, I almost forgot this PR and I am not very familiar with Github modifying the file directly.
I have put the tick back on `Create` and I want you to review it if you're free. Thanks.

**Comment by NelsonDane** [comment]
Unfortunately, I still got errors, but I was able to figure them out and update the instructions here (bf27e334806f32d8d2e39d5be570d7bf382aee75). If you come across a different issue or that wasn't the fix for you, then feel free to open/reopen a PR!

**Pull request closed without merging**